### ISSUE TITLE
Edit song/album/artist for YT videos

### ISSUE THREAD

**Issue by Monoxide8969** [25huizengek1/ViTune#308]
I'd like to be able to edit the song/album/artist labels (obv this just happens only locally and on frontend level) for youtube videos, which are usually in different formats, or the channel name is not the artist's and they obviously don't have any albums. This looks very messy and can make song difficult to find.

I think #210 needs to be sorted out first though, as that would make the song menus even bigger...

**Comment by 25huizengek1** [comment]
Yup, #210 has priority. Good call though, I mostly have songs from YouTube Music but I guess this makes sense for videos.
